samples/systemd.hcl

```diff
@@ -0,0 +1,5 @@
+systemd.unit "test" {
+  name   = "/tmp/foo8.service"
```

Comment:
could we have a more descriptive name here? Also, this seems to be an incomplete sample since the service file is undefined.